webapp/locales/en.json

```diff
@@ -82,8 +82,14 @@
     "download": {
       "name": "Download Data"
     },
-    "delete": {
-      "name": "Delete Account"
+    "deleteUserAccount": {
+      "name": "Delete Data",
+      "contributionsCount": "Delete my {count} posts",
+      "commentsCount": "Delete my {count} comments",
+      "accountDescription": "Be aware that your Post and Comments are important to our community. If you still choose to delete them, you have to mark them below.",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-07T22:27:55Z" title="Saturday, June 8th 2019, 12:27:55 am +02:00">Jun 8, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -74,7 +74,12 @@
       "name": "Download Data"
     },
     "delete": {
-      "name": "Delete Account"
+      "name": "Delete my User Account",
+      "countPosts": "Delete my {count} posts",
+      "countComments": "Delete my {count} comments",
+      "accountDescription": "Be aware that your Post and Comments are important to our community. If you still choose to delete those you have to mark them below.",
+      "accountWarning": "You <b>CAN'T MANAGE</b> and <b>CAN'T RECOVER</b> your Account, Posts... after deleting your account!",
+      "success": "Account successfully deleted"
```


Use https://www.deepl.com/translator for German translations and you will get an extra translation check from me for free :wink: